parts/appendices/B.fodt

```diff
@@ -13581,10 +13854,229 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
+          <text:p text:style-name="P1298">Avoid possible use of out-of-scope temporary (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5393" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5393</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">Fixed an issue where a UDQ assignment would not be performed  if it was triggered on the last time step of report step (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5379" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5379</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">Three small fixes: Make sure that the same explicit/implicit fraction is used when computing THP(BHP) and when computing BHP(THP). Reduced risk of switching control in case of extreme VFP-extrapolation. Disallow local well control switching when the maximum number of well switches has been reached (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5395" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5395</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">Fix the modification of transmissibility arrays (for example, using the MULTIPLY keyword in the EDIT section to modify TRANX) when running in parallel (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5414" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5414</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">Avoid recursively deleting results in all subfolders if the command line argument --output-dir is empty (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5427" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5427</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">Avoid mass creation when using DRSDT or DRSDTCON if hysteresis was deactivated (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5407" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5407</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">If convergence failures due to NaN then report this a problem rather than an error, then chop the time step and continue (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5434" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5434</text:a>). </text:p>
```

Comment:
```suggestion
          <text:p text:style-name="P1298">If convergence failures due to NaN then report this as a problem rather than an error, then chop the time step and continue (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5434" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5434</text:a>). </text:p>
```